### PR TITLE
Feature/165837 move between tenants

### PR DESCRIPTION
**Pull request by nicolasamaduro**
Add the disabled option on blip-selected

### CODE REVIEW

File: src/scss/components/_select.scss [hunk 1]
```diff
@@ -184,13 +184,15 @@
...
   -webkit-flex-wrap: nowrap;
 
-  &:hover,
-  &:focus {
-    background: $bp-color-whisper;
-    outline: 0;
-
-    .blip-select__option__label:not(:only-child) {
-      color: $bp-color-bot;
+  &:not(.blip-select__option-disabled) {
```
Comment (luizguicl) [inline src/scss/components/_select.scss]: Use BEM naming convention --disabled

File: src/scss/components/_select.scss [hunk 2]
```diff
@@ -259,6 +261,33 @@
...
....6022C16.4411 30.1334 17.0769 29.87 17.74 29.87H20.35V25.22C20.4545 21.0313 22.1919 17.0494 25.1914 14.1238C28.1909 11.1982 32.215 9.56073 36.405 9.56073C40.595 9.56073 44.6191 11.1982 47.6186 14.1238C50.6181 17.0494 52.3554 21.0313 52.46 25.22V29.87ZM28.7029 17.6152C26.6391 19.6179 25.4365 22.3456 25.35 25.22V29.87H47.46V25.22C47.3735 22.3456 46.1709 19.6179 44.1071 17.6152C42.0433 15.6126 39.2807 14.4925 36.405 14.4925C33.5293 14.4925 30.7666 15.6126 28.7029 17.6152ZM51.2654 56.1754C52.0943 55.3465 52.56 54.2223 52.56 53.05V34.87H20.24V53.05C20.2426 54.2232 20.7105 55.3474 21.541 56.176C22.3715 57.0047 23.4968 57.47 24.67 57.47H48.14C49.3122 57.47 50.4365 57.0043 51.2654 56.1754ZM38.7472 49.6837C38.0519 50.1462 37.2351 50.392 36.4 50.39C35.2825 50.3874 34.2117 49.9416 33.4225 49.1505C32.6332 48.3593 32.19 47.2875 32.19 46.17C32.19 45.3349 32.4378 44.5187 32.9019 43.8245C33.366 43.1303 34.0257 42.5894 34.7974 42.2703C35.5691 41.9512 36.4181 41.8682 37.237 42.0318C38.0558 42.1955 38.8078 42.5984 39.3975 43.1896C39.9873 43.7807 40.3884 44.5336 40.5501 45.3529C40.7118 46.1721 40.6268 47.0209 40.3059 47.7919C39.985 48.5628 39.4425 49.2212 38.7472 49.6837Z" fill="%23edf0ee"/> </svg>');
```
Comment (vinioo) [inline src/scss/components/_select.scss]: using just the relative path to the file didnt work?

Comment (nicolasamaduro) [reply src/scss/components/_select.scss]: yes, it didn't :( I'm going to create a stackoeverflow topic to explain it

**Pull request merged**